cmd/kyma/alpha/deploy/cmd.go

```diff
@@ -53,24 +53,40 @@ func NewCmd(o *Options) *cobra.Command {
 		RunE:    func(_ *cobra.Command, _ []string) error { return cmd.RunWithTimeout() },
 		Aliases: []string{"d"},
 	}
-	cobraCmd.Flags().StringArrayVarP(&o.Kustomizations, "kustomization", "k", []string{}, `Provide one or more kustomizations to deploy. Each occurrence of the flag accepts a URL with an optional reference (commit, branch, or release) in the format URL@ref or a local path to the directory of the kustomization file.
+	cobraCmd.Flags().StringArrayVarP(
+		&o.Kustomizations, "kustomization", "k", []string{},
+		`Provide one or more kustomizations to deploy. Each occurrence of the flag accepts a URL with an optional reference (commit, branch, or release) in the format URL@ref or a local path to the directory of the kustomization file.
 	Defaults to deploying Lifecycle Manager and Module Manager from GitHub main branch.
 	Examples:
 	- Deploy a specific release of the Lifecycle Manager: "kyma deploy -k https://github.com/kyma-project/lifecycle-manager/config/default@1.2.3"
 	- Deploy a local Module Manager: "kyma deploy --kustomization /path/to/repo/module-manager/config/default"
 	- Deploy a branch of Lifecycle Manager with a custom URL: "kyma deploy -k https://gitlab.com/forked-from-github/lifecycle-manager/config/default@feature-branch-1"
-	- Deploy the main branch of Lifecycle Manager while using local sources of Module Manager: "kyma deploy -k /path/to/repo/module-manager/config/default -k https://github.com/kyma-project/lifecycle-manager/config/default@main"`)
-	cobraCmd.Flags().StringArrayVarP(&o.Modules, "module", "m", []string{}, `Provide one or more modules to activate after the deployment is finished. Example: "--module name@namespace" (namespace is optional).`)
+	- Deploy the main branch of Lifecycle Manager while using local sources of Module Manager: "kyma deploy -k /path/to/repo/module-manager/config/default -k https://github.com/kyma-project/lifecycle-manager/config/default@main"`,
+	)
+	cobraCmd.Flags().StringArrayVarP(
+		&o.Modules, "module", "m", []string{},
+		`Provide one or more modules to activate after the deployment is finished. Example: "--module name@namespace" (namespace is optional).`,
+	)
 	cobraCmd.Flags().StringVarP(&o.ModulesFile, "modules-file", "f", "", `Path to file containing a list of modules.`)
-	cobraCmd.Flags().StringVarP(&o.Channel, "channel", "c", "stable", `Select which channel to deploy from: stable, fast, nightly.`)
+	cobraCmd.Flags().StringVarP(
+		&o.Channel, "channel", "c", "regular", `Select which channel to deploy from.`,
+	)
+	cobraCmd.Flags().StringVarP(
+		&o.Namespace, "namespace", "n", "kyma-system",
+		"The namespace to deploy the Kyma custom resource in. An empty namespace uses 'kyma-system'",
```

Comment:
The default value message will be added by cobra, no need to mention it. 
```suggestion
		"The namespace to deploy the Kyma custom resource in",
```